host/crypto/bcrypt/cert.c

```diff
@@ -1296,3 +1296,39 @@ oe_result_t oe_cert_find_extension(
 done:
     return result;
 }
+
+oe_result_t oe_cert_get_validity_dates(
+    const oe_cert_t* cert,
+    oe_datetime_t* not_before,
+    oe_datetime_t* not_after)
+{
+    oe_result_t result = OE_UNEXPECTED;
+    const cert_t* impl = (const cert_t*)cert;
+
+    if (not_before)
```

Comment:
Same comment on the combining not_before blocks

Reply:
Done.